.bin/cleanup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set +ex
+SITE_ENV="${TERMINUS_SITE}.ci-${BUILD_NUM}"
+# Nuke any lingering multidev environments from orbit.
```

Comment:
This means multiple rounds of testing will get in each others way, or kill a multidev while another branch/build/whatever is in-progress. Alternative will be to cleanup only _this build_ and any build older than X days, and separately have a script  that deletes the oldest multidev at the _top_ of the CI run. I did this for a Drupal module somewhere and we could dig that up. Probably a pretty good custom action tbh.

Reply:
https://github.com/pantheon-systems/pantheon_advanced_page_cache/pull/51/files#diff-c08821ed2397f2d1cb87ba9b78650da7f45812c1237a699af51be8776a8b8d55

Reply:
"any build older than X days" should be a cron task, not part of any on-push task or anything like that, to keep time of test iteration down.

Reply:
we can pull this out since the cleanup step is always being run. there are a lot of other places where we don't actually clean up after failed tests.